migrations/20241016074609_get_ready_for_dummy_proof_dispatch_l1_batches.sql

```diff
@@ -0,0 +1 @@
+-- Add migration script here
```

Comment:
there's no migration, is it right?

Reply:
Oh, thanks for mentioning it
I removed it